crates/wash-cli/src/up/mod.rs

```diff
@@ -66,6 +66,14 @@ pub struct NatsOpts {
         requires = "nats_remote_url"
     )]
     pub nats_credsfile: Option<PathBuf>,
+    /// Optional path to a NATS configuration file | NOTE: Works only if accompanied by nats-host and nats-port
+    #[clap(
+        long = "nats-config-dir",
+        env = "NATS_CONFIG",
+        requires = "nats_host",
+        requires = "nats_port"
+    )]
+    pub nats_configfile_dir: Option<PathBuf>,
```

Comment:
```suggestion
    pub nats_config_dir: Option<PathBuf>,
```

This change will break other stuff, but we don't need "configfile" right? 

Also, please note in the `///` comment that it is the path to a NATS config folder (i.e. one that contains a NATS config file -- named <file name>)